panel/src/components/Collection/Items.vue

```diff
@@ -7,7 +7,10 @@
 		@sort="$emit('sort', $event)"
 		@option="onOption"
 	>
-		<template #options="{ row: item, rowIndex: index }">
+		<template
+			v-if="$scopedSlots.options"
+			#options="{ row: item, rowIndex: index }"
+		>
```

Comment:
Can you give an example when this is needed? Isn't the slot only filled anyways if provided?